druntime/src/importc.h

```diff
@@ -69,14 +69,26 @@
 /********************************
  * __has_extension is a clang thing:
  *    https://clang.llvm.org/docs/LanguageExtensions.html
- * ImportC doesn't has those extensions.
+ * ImportC no has extensions.
```

Comment:
Can has sensical grammar? 
```suggestion
 * ImportC has no has extensions.
```

Reply:
One negation is all that's necessary!

Reply:
Yes, but it should be "ImportC **has no** extensions."

Reply:
???
Has is a verb (to possess), not a negation before the no, and `__has` is a singular pronoun, not a negation of the previous quantifier.

Reply:
"no has extensions" is just continuing the joke of "gcc has extension".